test/validate/linux_validate.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 var (
-	restartNetworkCmd      = []string{"bash", "-c", "chroot /host /bin/bash -c systemctl restart systemd-networkd"}
+	restartNetworkCmd      = []string{"bash", "-c", "systemctl restart systemd-networkd"}
```

Comment:
Did we confirm this works ? I remember the chroot was added because by default we cannot restart the s`ystemd-network`

Reply:
Yes, I confirmed that it restarts properly. Feel free to reconfirm.